scraper/scraper/spiders/h1Spider.py

```diff
@@ -0,0 +1,18 @@
+from scrapy.selector import Selector
+from scrapy.http import HtmlResponse
+import scrapy
+
+
+class QuotesSpider(scrapy.Spider):
```

Comment:
I believe the `QuotesSpider` name is because you use [scrapy tutorial](https://doc.scrapy.org/en/latest/intro/tutorial.html), I think it should be called `PensionSpider` or something.